Networking/Networking/Requests/JetpackRequest.swift

```diff
@@ -28,7 +28,7 @@ struct JetpackRequest: URLRequestConvertible {
 
     /// Jetpack-Tunneled Parameters
     ///
-    let parameters: [String: String]
+    let parameters: [String: Any]
```

Comment:
This has to do with adding a POST request. We're now able to pass complete Codable models to the API.

Reply:
hmm I thought we had POST requests before? just wondering what's different in the API requests in this PR that prompted this change

Reply:
Now `JetpackRequest` accept a dictionary `[String: Any]` as parameters because in this way we can manage API requests with a nested body, like the creation of a new refund which needs a body like this:
```
{
	"amount": "{{amount}}",
	"reason": "{{optional_refund_reason}}",
	"line_items": {
		"{{line_item_id}}": {
			"qty": "{{quantity_of_product_to_refund}}",
			"refund_total": "{{product_price_without_tax}}"
		}
	}
}
```

Reply:
For example, it will be much easier to pass in an `OrderRefund` object or `Refund` object. Granted, `OrderRefund` needs to become a `Codable` struct instead of only following the `Decodable` protocol. I can do that part when I work on Refunds Step 2.